src/main/java/Lesson18/OrderDaoImpl.java

```diff
@@ -44,39 +45,101 @@ private static SessionFactory buildSessionFactory() {
 
     @Override
     public Set<Order> getAllOrders() {
-        return null;
+        Set<Order> orders = new HashSet<>();
+        try {
+            sessionObj = buildSessionFactory().openSession();
+            sessionObj.beginTransaction();
+
+            orders = new HashSet<>(sessionObj.createQuery("FROM Order").getResultList());
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        } finally {
+            sessionObj.close();
```

Comment:
if you have exception on creation session your sessionObj will be null. And you will have NullPointerException